### PR TITLE
docs:  tables info popover styles

### PR DESCRIPTION
**Pull request by tristandubbeld**
I was browsing the docs in a bright environment with light mode turned on and noticed the popovers weren't readable. In this small contribution colors in light mode are fixed. I've also updated the positioning of the popover trigger in the `EmitsTable` to match the `PropsTable`.

### CODE REVIEW

File: docs/components/tables/EmitsTable.vue [hunk 1]
```diff
@@ -32,23 +32,25 @@ const props = defineProps<EmitsTableProps>()
...
-              <PopoverContent
-                side="top"
-                class="text-xs rounded-lg px-3 py-2.5 bg-neutral-800 w-[265px] shadow-[0_10px_38px_-10px_hsla(206,22%,7%,.35),0_10px_20px_-15px_hsla(206,22%,7%,.2)] focus:shadow-[0_10px_38px_-10px_hsla(206,22%,7%,.35),0_10px_20px_-15px_hsla(206,22%,7%,.2),0_0_0_2px_theme(colors.violet7)] will-change-[transform,opacity] data-[state=open]:data-[side=top]:animate-slideDownAndFade data-[state=open]:data-[side=right]:animate-slideLeftAndFade data-[state=open]:data-[side=bottom]:animate-slideUpAndFade data-[state=open]:data-[side=left]:animate-slideRightAndFade"
-              >
-                <span size="2" class="leading-5" v-html="prop.description" />
-                <PopoverArrow class="fill-neutral-800" :size="8" />
-              </PopoverContent>
-            </PopoverRoot>
-          </template>
+          <div class="flex items-center gap-1">
```
Comment (tristandubbeld) [inline docs/components/tables/EmitsTable.vue:35]: Added the `flex` `div` to align the text with the popover trigger.

File: docs/components/tables/EmitsTable.vue [hunk 2]
```diff
@@ -32,23 +32,25 @@ const props = defineProps<EmitsTableProps>()
...
+            <ProseCodeInline class=" !text-[13px]">
+              {{ prop.name }}{{ prop.required ? "*" : null }}
+            </ProseCodeInline>
+            <template v-if="prop.description">
+              <PopoverRoot>
+                <PopoverTrigger class="p-1.5 rounded hover:bg-neutral-100 dark:hover:bg-neutral-800 focus:outline-2 outline-neutral-800">
```
Comment (tristandubbeld) [inline docs/components/tables/EmitsTable.vue:41]: Updated the hover styles (same as in `PropsTable`).

File: docs/components/tables/EmitsTable.vue [hunk 3]
```diff
@@ -32,23 +32,25 @@ const props = defineProps<EmitsTableProps>()
...
+                  <Icon icon="radix-icons:info-circled" class="h-4 w-4 text-neutral-400" />
+                </PopoverTrigger>
+                <PopoverContent
+                  side="top"
+                  class="text-xs rounded-lg px-3 py-2.5 bg-white dark:bg-neutral-800 w-[265px] shadow-[0_10px_38px_-10px_hsla(206,22%,7%,.35),0_10px_20px_-15px_hsla(206,22%,7%,.2)] focus:shadow-[0_10px_38px_-10px_hsla(206,22%,7%,.35),0_10px_20px_-15px_hsla(206,22%,7%,.2),0_0_0_2px_theme(colors.violet7)] will-change-[transform,opacity] data-[state=open]:data-[side=top]:animate-slideDownAndFade data-[state=open]:data-[side=right]:animate-slideLeftAndFade data-[state=open]:data-[side=bottom]:animate-slideUpAndFade data-[state=open]:data-[side=left]:animate-slideRightAndFade"
```
Comment (tristandubbeld) [inline docs/components/tables/EmitsTable.vue:46]: Updated the background with `bg-white dark:bg-neutral-800`.

File: docs/components/tables/EmitsTable.vue [hunk 4]
```diff
@@ -32,23 +32,25 @@ const props = defineProps<EmitsTableProps>()
...
+                >
+                  <span size="2" class="leading-5" v-html="prop.description" />
+                  <PopoverArrow class="dark:fill-neutral-800 fill-white" :size="8" />
```
Comment (tristandubbeld) [inline docs/components/tables/EmitsTable.vue:49]: Added fill for dark and light mode.

**Review by tristandubbeld — COMMENTED** [review]
I've added some comments to the `EmitsTable` to make it a bit clearer what has changed. The indentation changes make it a bit unclear.

**Review by zernonia — APPROVED** [review]
lgtm! Thanks @tristandubbeld

**Pull request merged**